### PR TITLE
Add Dockerfile

### PR DESCRIPTION
**Pull request by Zelldon**
Add first step to create docker image 
I need to automate the docker image publishing but for now it should be ok it is published on https://hub.docker.com/repository/docker/zelldon/zeebe-hazelcast

This makes it possible to test the c# connector with test container etc.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -127,6 +127,30 @@ The exporter and the Java connector can be built with Maven
 
 `mvn clean install`
 
+## Build Docker image
+
+The docker image can build like this:
+
+```
+docker build --build-arg EXPORTERJAR=exporter/target/zeebe-hazelcast-exporter-0.8.0-SNAPSHOT-jar-with-dependencies.jar .
```
Comment (saig0) [inline README.md]:
```suggestion
docker build --build-arg EXPORTERJAR=exporter/target/zeebe-hazelcast-exporter-0.8.0-alpha1-jar-with-dependencies.jar .
```

File: README.md [hunk 2]
```diff
@@ -127,6 +127,30 @@ The exporter and the Java connector can be built with Maven
...
+```
+
+The latest image is also published under https://hub.docker.com/repository/docker/zelldon/zeebe-hazelcast
+
+You can just run:
+
+```
+docker run zelldon/zeebe-hazelcast:TAG
+```
```
Comment (saig0) [inline README.md]: I think we can move this part in the `Install` section as easy alternative to the manual setup :)

File: README.md [hunk 3]
```diff
@@ -127,6 +127,30 @@ The exporter and the Java connector can be built with Maven
...
+
+To publish the latest version:
+
+```
+docker build --build-arg EXPORTERJAR=exporter/target/zeebe-hazelcast-exporter-0.8.0-SNAPSHOT-jar-with-dependencies.jar -t <username>/repo:TAG .
```
Comment (saig0) [inline README.md]:
```suggestion
docker build --build-arg EXPORTERJAR=exporter/target/zeebe-hazelcast-exporter-0.8.0-alpha1-jar-with-dependencies.jar -t <username>/repo:TAG .
```

File: zeebe.cfg.toml [hunk 4]
```diff
@@ -0,0 +1,323 @@
...
+className = "io.zeebe.hazelcast.exporter.HazelcastExporter"
+
+  [exporters.args]
+  # comma separated list of io.zeebe.protocol.record.ValueType
+  enabledValueTypes = "JOB,WORKFLOW_INSTANCE,DEPLOYMENT,INCIDENT,TIMER,VARIABLE,MESSAGE,MESSAGE_SUBSCRIPTION,MESSAGE_START_EVENT_SUBSCRIPTION"
+
+   # If true, the exporter update its position after publish the record to Hazelcast.
+   # Otherwise, it never update its position. On broker start, it will always start from the begin of the log.
+   # CAUTION! The broker can't delete data and may run out of disk space if set to false.
+   updatePosition = false
```
Comment (saig0) [inline zeebe.cfg.toml]: The `updatePosition` configuration is not used anymore since we switched to ringbuffer.

File: README.md (hunk 2 above)
Comment (Zelldon) [reply README.md]: :+1:

File: zeebe.cfg.toml (hunk 4 above)
Comment (Zelldon) [reply zeebe.cfg.toml]: I just copied the cfg from simple monitor :smile:

**Comment by Zelldon** [comment]
> I think we can automate the Docker build and push either using the Zeebe CI or Github Docker actions.

Agree but had no time yesterday :) I will do when I have more time

**Pull request merged**